spec/models/post_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+  subject { user.posts.create(title: 'Coding in progress', text: 'One line at a time') }
+
+  describe 'validations' do
+    it 'should have a title' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'should have text' do
+      subject.text = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'title should not be greater than 250' do
+      subject.title = 'Hello world' * 25
+      expect(subject).to_not be_valid
+    end
+
+    it 'comments_counter should be an integer' do
+      subject.comments_counter = ''
+      expect(subject).to_not be_valid
+    end
+
+    it 'likes_counter should be an integer' do
+      subject.likes_counter = ''
+      expect(subject).to_not be_valid
+    end
+
+    it 'comments_counter should be greater than or equal to 0' do
+      subject.comments_counter = -1
+      expect(subject).to_not be_valid
+    end
+
+    it 'likes_counter should be greater than or equal to 0' do
+      subject.likes_counter = -1
+      expect(subject).to_not be_valid
+    end
+  end
+
```

Comment:
- [ ] Great job including all the requested tests :clap: However, I noticed that all of your tests are currently failing. So kindly fix those so that all your tests can pass as expected :100: 
- Reference photo
![image](https://github.com/Roland-Ntwali/Blog-App/assets/102699958/5b65e641-eecb-466b-a261-4f97a00053ee)